pkg/networkmanager/firewall.jsx

```diff
@@ -140,16 +136,45 @@ function ZoneRow(props) {
         deleteButton = <button className="btn btn-danger pficon pficon-delete" onClick={onRemoveZone} />;
     }
 
-    const columns = [
-        { name: props.zone.name, header: true },
-        <>{ props.zone.id === firewall.defaultZone ? <><span className="fa fa-check" /> default</> : '' }</>,
-        <>{ props.zone.interfaces.length > 0 ? props.zone.interfaces.join(', ') : '*' }</>,
-        <>{ props.zone.source.length > 0 ? props.zone.source.join(', ') : '*' }</>,
-        deleteButton,
-    ];
-    return <ListingRow key={props.zone.id}
-                       rowId={props.zone.id}
-                       columns={columns} />;
+    var addServiceAction;
+    if (firewall.readonly) {
+        addServiceAction = (
+            <OverlayTrigger placement="top"
+                                overlay={ <Tooltip id="tip-auth">{ _("You are not authorized to modify the firewall.") }</Tooltip> }>
+                <Button bsStyle="primary" className="pull-right" disabled> {_("Add Services")} </Button>
+            </OverlayTrigger>
+        );
+    } else {
+        addServiceAction = (
+            <Button bsStyle="primary" onClick={() => props.openServicesDialog(props.zone.id, props.zone.name)} id="add-services-button">
+                {_("Add Services")}
+            </Button>
+        );
+    }
+
+    return <div className="zone-section" data-id={props.zone.id}>
+        <h4>
+            <span>
+                <strong>{ cockpit.format(_("$0 Zone"), props.zone.name) }</strong>
+                { props.zone.interfaces.length > 0 && <span>&nbsp;&nbsp;&nbsp;<strong>{_("Interfaces")}</strong> {props.zone.interfaces.join(", ")}</span> }
```

Comment:
could this spacing be rather done with a bit of css? (`.zone-section h3 strong` could be selector and `padding-left: 1px` could be the rule)